### PR TITLE
Fix code examples for doctests

### PR DESCRIPTION
**Pull request by stevhliu**
This PR fixes some code examples to pass the doctests for the pipeline and `AutoClass` tutorials. 

I was unable to pass the audio code examples on my local machine because soundfile is not supported on M1 yet. I was able to run and reproduce the code snippets in Colab though so I think they should also pass on the CI.

### CODE REVIEW

File: docs/source/en/pipeline_tutorial.mdx [hunk 1]
```diff
@@ -126,14 +143,10 @@ Specify your vision task and pass your image to the classifier. The imaage can b
...
- {'label': 'cougar, puma, catamount, mountain lion, painter, panther, Felis concolor',
-  'score': 0.03433405980467796},
- {'label': 'snow leopard, ounce, Panthera uncia',
-  'score': 0.032148055732250214},
- {'label': 'Egyptian cat', 'score': 0.02353910356760025},
- {'label': 'tiger cat', 'score': 0.023034192621707916}]
-```
+>>> preds = [{"score": round(pred["score"], 4), "label": pred["label"]} for pred in preds]
+>>> print(preds)
+[{'score': 0.4403, 'label': 'lynx, catamount'}, {'score': 0.0343, 'label': 'cougar, puma, catamount, mountain lion, painter, panther, Felis concolor'}, {'score': 0.0321, 'label': 'snow leopard, ounce, Panthera uncia'}, {'score': 0.0235, 'label': 'Egyptian cat'}, {'score': 0.023, 'label': 'tiger cat'}]
```
Comment (sgugger) [inline docs/source/en/pipeline_tutorial.mdx:151]: Can we show the results as above? The doc test will pass if your replace any space by a new line, it only cares that there is whitespace.

File: docs/source/en/pipeline_tutorial.mdx [hunk 2]
```diff
@@ -39,7 +39,9 @@ While each task has an associated [`pipeline`], it is simpler to use the general
 2. Pass your input text to the [`pipeline`]:
 
 ```py
->>> generator("Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone")
+>>> generator(
+...     "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone"
+>>> )  # doctest: +SKIP
```
Comment (sgugger) [inline docs/source/en/pipeline_tutorial.mdx:44]: Thos `>>>` will be fixed very soon, if we can merge after the fix is out and `doc-builder` gets a new release, that would be great!

**Review by sgugger — APPROVED** [review]
Thanks for fixing!

**Comment by ydshieh** [comment]
Thanks for the work.

Other than the `>>>` things, there is one example failed.

For audio pipeline:
```
Expected:
    [{'score': 0.4403, 'label': 'lynx, catamount'}, {'score': 0.0343, 'label': 'cougar, puma, catamount, mountain lion, painter, panther, Felis concolor'}, {'score': 0.0321, 'label': 'snow leopard, ounce, Panthera uncia'}, {'score': 0.0235, 'label': 'Egyptian cat'}, {'score': 0.023, 'label': 'tiger cat'}]
Got:
    [{'score': 0.4335, 'label': 'lynx, catamount'}, {'score': 0.0348, 'label': 'cougar, puma, catamount, mountain lion, painter, panther, Felis concolor'}, {'score': 0.0324, 'label': 'snow leopard, ounce, Panthera uncia'}, {'score': 0.0239, 'label': 'Egyptian cat'}, {'score': 0.0229, 'label': 'tiger cat'}]
```
(this is just a format issue I think)

For vision pipeline:
```
Expected:
    [{'score': 0.4403, 'label': 'lynx, catamount'}, {'score': 0.0343, 'label': 'cougar, puma, catamount, mountain lion, painter, panther, Felis concolor'}, {'score': 0.0321, 'label': 'snow leopard, ounce, Panthera uncia'}, {'score': 0.0235, 'label': 'Egyptian cat'}, {'score': 0.023, 'label': 'tiger cat'}]
Got:
    [{'score': 0.4335, 'label': 'lynx, catamount'}, {'score': 0.0348, 'label': 'cougar, puma, catamount, mountain lion, painter, panther, Felis concolor'}, {'score': 0.0324, 'label': 'snow leopard, ounce, Panthera uncia'}, {'score': 0.0239, 'label': 'Egyptian cat'}, {'score': 0.0229, 'label': 'tiger cat'}]
```
(This might be due to some random ops. I remembered I have similar situations before. I can take a look too.)

**Comment by sgugger** [comment]
I don't know why you ask me @ydshieh this is not my PR ;-)

**Comment by ydshieh** [comment]
> I don't know why you ask me @ydshieh this is not my PR ;-)

I know. Just to make sure you are also fine with my suggestion about `just update the values`. But I guess I should be more confident 😄

**Pull request closed without merging**